x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_bulk_to_timeline.tsx

```diff
@@ -241,7 +241,7 @@ export const useAddBulkToTimelineAction = ({
        * */
       const onResponseHandler = (localResponse: TimelineArgs) => {
         sendBulkEventsToTimelineHandler(localResponse.events);
-        if (tableId === TableId.alertsOnAlertsPage) {
+        if (tableId === TableId.alertsOnAlertsPage || tableId === TableId.alertsOnAttacksPage) {
```

Comment:
Nit: Maybe do `DETECTIONS_TABLE_IDS.some((tableId) => tableId === tableType)` like in the other places.

Reply:
Actually, nvm. `DETECTIONS_TABLE_IDS` has more than 2 elements.